README.md

```diff
@@ -5,14 +5,25 @@ Docusaurus website serving to showcase the AI Lab's activities.
 https://ai-cfia.github.io/ai-cfia-ia-acia.github.io/
 
 ## Developing Locally
-After opening the project in the devcontainer, change your current working directory:  
+After opening the project in the devcontainer, change your current working directory:
 
 `cd ai-cfia-ia-acia`
 
 Then install the npm dependencies:
 
 `npm install`
 
-You can now run the development server: 
+You can now run the development server:
 
-`npm start -- --port 3001`
\ No newline at end of file
+`npm start -- --port 3001`
+
+## Deployment
+The website is automatically deployed to GitHub Pages through a streamlined process that begins with a single trigger:
+
+When a push is made to the `main` branch, it initiates the `Deploy to GitHub Pages` workflow. This singular action sets off a series of automated steps:
+
+- **Build Process**: Static assets are generated during the workflow run.
+- **Permissions**: Write permissions to the repository contents are verified to enable updates to GitHub Pages.
+- **Execution of Deployment Job**: Utilizing a reusable workflow from the [workflow repository](https://github.com/ai-cfia/github-workflows), the job configures the environment with the specified Node.js version, sets the working directory, identifies the directory from which to publish, and designates the `deployment` branch as the publication target.
```

Comment:
this came up during ADR and should apply to all our markdown docs: please format to wrap at 80.

also, instead of one long run on sentence, break it down with sub-bullet points:

- **Execution of Deployment Job**: Utilizing a reusable workflow from the [workflow repository](https://github.com/ai-cfia/github-workflows), the job :
   - configures the environment with the specified Node.js version
   - sets the working directory
   -  identifies the directory from which to publish
   - designates the `deployment` branch as the publication target